src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -87,39 +87,41 @@ public static function substr(?string $string, int $from, ?int $length = null):
 
     public static function formatTime(int|float $secs, int $precision = 1): string
     {
+        $ms = $secs * 1000;
```

Comment:
Rounding to int is necessary for modulo to work correctly.

As you don't use sub millisecond time, keeping the decimals is pointless.